main.js

```diff
@@ -135,11 +135,14 @@ fs.readFile(`${pathRawDetectionsInput}`, function (err, f) {
         var detectionOfThisFrameArray = line.split(",");
         var detectionFrameIndex = parseInt(detectionOfThisFrameArray[0], 10);
         if (!Number.isNaN(detectionFrameIndex)) {
+          var w = parseFloat(detectionOfThisFrameArray[4]);
+          var h = parseFloat(detectionOfThisFrameArray[5]);
+
           var detection = {
-            x: parseFloat(detectionOfThisFrameArray[2]),
-            y: parseFloat(detectionOfThisFrameArray[3]),
-            w: parseFloat(detectionOfThisFrameArray[4]),
-            h: parseFloat(detectionOfThisFrameArray[5]),
+            x: parseFloat(detectionOfThisFrameArray[2]) + w / 2,
+            y: parseFloat(detectionOfThisFrameArray[3]) + h / 2,
```

Comment:
Shouldn't we subtract the width and height instead of adding it? Or am I misunderstanding the coordinate system?

Reply:
It should be correct that way. The format of the MOT17 input file is
`<frame>, <id>, <bb_left>, <bb_top>, <bb_width>, <bb_height>, <conf>, <x>, <y>, <z>`
so we're going from top left corner to center point, which means adding half of width and height.

Reply:
Sorry, you're right! 👍